okta-hosted-login/README.md

```diff
@@ -14,9 +14,20 @@ Before running this sample, you will need the following:
 
 Clone this repo and replace the okta configuration placeholders in the `appsettings.json` with your configuration values from the Okta Developer Console. 
 You can see all the available configuration options in the [okta-aspnet GitHub](https://github.com/okta/okta-aspnet/blob/master/README.md).
-For step-by-step instructions, visit the Okta [ASP.NET MVC quickstart]. The quickstart will guide you through adding Okta login to your ASP.NET application.
+For step-by-step instructions, visit the Okta [Sign Users in to Your Web Application guide]. The guide will walk you through adding Okta login to your ASP.NET application.
 
-Now start your server and navigate to http://localhost:8080 in your browser.
```

Comment:
I think we should take 1 more pass at this readme to make it even clearer. If we took a step back and looked at the steps a developer would need to do starting from "clone", what do those look like? Is it

1. Clone repo
2. Run the project, figure out which port it's running on
2a. Trust the local dev certificate if necessary
3. Add the correct configuration to the Okta Developer Console (including the port you just found)
4. Add that same config to the sample's appconfig
5. Run again (in an incognito window for best results)

Reply:
I like this approach. I updated the readme following this structure. Once we are OK with it, I'll update the other readme files.